### PR TITLE
DSD-1266 & DSD-1275: new pages for A11y Guide

### PR DESCRIPTION
**Pull request by bigfishdesign13**
Fixes JIRA ticket [DSD-1266](https://jira.nypl.org/browse/DSD-1266) & [DSD-1275](https://jira.nypl.org/browse/DSD-1275)

## This PR does the following:

- Adds docs for `Links` and `Repetitive Actions` to the `Accessibility Guide`.

## How has this been tested?

- local Storybook

## Accessibility concerns or updates

- n/a

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/components/AccessibilityGuide/RepetitiveActions.stories.mdx
Comment (EdwinGuzman) [inline src/components/AccessibilityGuide/RepetitiveActions.stories.mdx]: Minor typo with `<?span>`.

File: src/components/AccessibilityGuide/RepetitiveActions.stories.mdx
Comment (EdwinGuzman) [inline src/components/AccessibilityGuide/RepetitiveActions.stories.mdx:54]: Should we include a snippet of CSS that shows what the `sr-only` class can be?

**Review by EdwinGuzman — APPROVED** [review]
Just minor comments but this is great. Approving because this still seems like a WIP given the "TBD" sections.

File: src/components/AccessibilityGuide/RepetitiveActions.stories.mdx
Comment (bigfishdesign13) [reply src/components/AccessibilityGuide/RepetitiveActions.stories.mdx:54]: Yes, that is a good idea.

**Comment by bigfishdesign13** [comment]
> Just minor comments but this is great. Approving because this still seems like a WIP given the "TBD" sections.

@EdwinGuzman 
Yes, it is kind of like a work in progress.  We still need to make some decisions about our patterns, but I wanted to get the initial information out there.  Would you prefer I wait until we have the patterns etablished before I merge this?

**Review by EdwinGuzman — APPROVED** [review]
I don't think we need to wait to get the NYPL patterns for this to go out.

**Pull request merged**
